bftengine/src/bftengine/Bitmap.hpp

```diff
@@ -41,6 +41,18 @@ class Bitmap {
     }
   }
 
+  Bitmap(Bitmap&& other) : numBits_(other.numBits_), p_(nullptr) {
+    if (numBits_ == 0) {
```

Comment:
This if statement is a little strange to me. Either we assume `numBits_ == 0 && `p_ == nullptr` or otherwise this move constructor can trigger leaks if `p != nullptr`. If it must always be nullptr, as I think it should in this case, we can just skip the if altogether and assign directly. We can possibly throw an assert in for good measure.

Reply:
Yes, will add proper Assertions and remove the if.